src/main/java/Ui.java

```diff
@@ -0,0 +1,39 @@
+import java.util.Scanner;
+
+public class Ui {
+    private Scanner in = new Scanner(System.in);
+    public void printlogo(){
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+    }
+    public void greetInfo() {
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+    public void exitInfo(){
+        System.out.println("----------------\n"+"Bye. Hope to see you again soon!");
+    }
+    public String getInput(Scanner in) {
+        return in.nextLine().trim();
+    }
+    public void printList(Task [] tasks,int size){
+        System.out.println("-----------");
+        for (int i = 0;i<size;i++){
+            System.out.println((i+1)+"."+ "["+tasks[i].getStatusIcon()+"] "+
+                    tasks[i].description);
+        }
+        System.out.println("-----------");
+    }
+    public void printTask(String str,int count){
+        System.out.println("--------------\nGot it. I've added this task: " + "\n"+str);
+        System.out.println("Now you have "+count+" in the list.\n-------------------");
+    }
+    public void printMsg(Task t){
```

Comment:
printMsg sounds quite general. Should we make it `printSetDoneResult`? But this will definitely increase the length of the function name, haha.